packages/open-ui/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { MarginStyles } from 'interfaces/MarginStyles';
+import { Title } from 'components/Typography';
+import { verticalRhythm } from 'constants/styles';
+import styles from './Card.module.scss';
+
+interface CardProps {
+  title?: string;
+}
+
+export const Card: React.FC<CardProps & MarginStyles> = props => {
+  const { children, title, ...styleProps } = props;
```

Comment:
@remotealex should consider renaming it to marginStyles ? Cause later on we might can consider adding the props.

Reply:
Yeah, good shout